extensions/amp-ad-network-doubleclick-impl/0.1/test/test-amp-ad-network-doubleclick-impl.js

```diff
@@ -810,6 +810,36 @@ describes.realWin('amp-ad-network-doubleclick-impl', realWinConfig, env => {
       impl.getAdUrl(CONSENT_POLICY_STATE.UNKNOWN_NOT_REQUIRED).then(url => {
         expect(url).to.not.match(/(\?|&)npa=(&|$)/);
       }));
+
+    it('should include msz/psz if in experiment', () => {
+      sandbox.stub(impl, 'randomlySelectUnsetExperiments_').returns(
+          {flexAdSlots: '21063174'});
+      impl.setPageLevelExperiments();
+      return impl.getAdUrl().then(url => {
+        expect(url).to.match(/(\?|&)msz=[0-9]+x-1(&|$)/);
+        expect(url).to.match(/(\?|&)psz=[0-9]+x-1(&|$)/);
+        expect(url).to.match(/(=|%2C)21063174(%2C|&|$)/);
+      });
+    });
+
+    it('should not include msz/psz if not in flexAdSlots control', () => {
+      sandbox.stub(impl, 'randomlySelectUnsetExperiments_').returns(
+          {flexAdSlots: '21063173'});
+      impl.setPageLevelExperiments();
+      return impl.getAdUrl().then(url => {
+        expect(url).to.not.match(/(\?|&)msz=[0-9]+x-1(&|$)/);
+        expect(url).to.not.match(/(\?|&)psz=[0-9]+x-1(&|$)/);
+        expect(url).to.match(/(=|%2C)21063173(%2C|&|$)/);
+      });
+    });
+
+    it('should not include msz/psz if not in flexAdSlots experiment', () => {
+      return impl.getAdUrl().then(url => {
+        expect(url).to.not.match(/(\?|&)msz=[0-9]+x-1(&|$)/);
+        expect(url).to.not.match(/(\?|&)psz=[0-9]+x-1(&|$)/);
+        expect(url).to.not.match(/(=|%2C)2106317(3|4)(%2C|&|$)/);
```

Comment:
Here and above, can we assert "no msz and psz params" instead of "no valid params"?

Reply:
Done.